tests/test_skeleton.py

```diff
@@ -0,0 +1,56 @@
+"""
+Base test skeleton.
+"""
+
+import os
+import sys
+import unittest
+
+from flask.ext.testing import TestCase, LiveServerTestCase
+
+from app import app, db
+
+class TestSkeleton(TestCase):
+
+    # Required for flask-testing
+    # See https://pythonhosted.org/Flask-Testing/#testing-with-liveserver
+    def create_app(self):
+        return app
+
+
+    @classmethod
+    def setUpClass(cls):
+        print "Setting up"
```

Comment:
This part makes the tests look a bit different from traditional unit tests:
```
Setting up
..Setting up
F...FSetting up
FF...FF
```
Normally it would be a series of dots and letters. I'd take this part out.